local.go

```diff
@@ -50,6 +50,14 @@ func (r *LocalRepository) Resolve(ref *charm.Reference) (*charm.URL, error) {
 	if ref.Revision != -1 {
 		return u, nil
 	}
+	if ref.Series == "bundle" {
+		if _, err = r.GetBundle(u); err != nil {
```

Comment:
I don't think we need to do this. We know the only possible revision and that's good enough for Resolve.